app/pods/application/route.js

```diff
@@ -20,9 +20,8 @@ export default class ApplicationRoute extends Route {
   }
 
   async beforeModel() {
-    this.moment.setLocale('nl');
-    this.moment.defaultFormat = 'DD.MM.YYYY';
-    this.moment.allowEmpty = true;
```

Comment:
Guessing we don't need `allowEmpty` here since it is already in the environment config?

Reply:
Indeed